stdlib/REPL/src/LineEdit.jl

```diff
@@ -1979,11 +1979,11 @@ function enter_prefix_search(s::MIState, p::PrefixHistoryPrompt, backward::Bool)
         pss.indent = state(s, parent).indent
         pss.mi = s
     end
-    pss = state(s, p)
+    pss_ = state(s, p)
```

Comment:
Maybe we can mark either of `pss` here or on L1974?

Reply:
By "mark" do you mean adding `local pss = ...`?

Reply:
Ah, yes.

Reply:
L1974 would make no difference (it is already the local decl)